stddev strat updated

TheRoboKitten opened the pull request:
stddev strat updated

This one works great, 
Removed index.php.
Removed unused package.json item.

stats-lite is required for generating stddev though

Commit:
Create strategy.js

Commit:
Create _codemap.js

Commit:
Update package.json

Commit:
Create index.php

Commit:
Update strategy.js

Commit:
Update strategy.js

Commit:
Update strategy.js

Commit:
Updated

Commit:
Update strategy.js

Commit:
Update strategy.js

Commit:
Delete index.php

Commit:
Update package.json

Commit:
best one so far

Commit:
Update strategy.js

Commit:
Update strategy.js

jessecurry commented on extensions/strategies/stddev/strategy.js:
```diff
@@ -0,0 +1,53 @@
+    getOptions: function () {
+      this.option('period', 'period length', String, '100ms')
+      this.option('trendtrades_1', "Trades for trend 1", Number, 100)
+      this.option('trendtrades_2', "Trades for trend 1", Number, 300)
```
Should this be: `this.option('trendtrades_2', "Trades for trend 2", Number, 300)`?

JensvdHeydt commented:
Is this actually finished? We startet to review though you seem to constantly push commits... :)

Commit:
Create backtest.py

Commit:
Update strategy.js

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Update strategy.js

Commit:
Tune sell order first, then buy order if trend/graph starts on UP signal.

Tune sell order first, then buy order if trend/graph starts on UP signal.
Look at the graph and results, check when it buys and sells.

If signal is bought in beginning (on up) you need to make sure the sell signal is hit as soon as it becomes the first amount of sensitive to tune the standard deviation sell order.

Commit:
Update strategy.js

Commit:
Update backtest.py

Commit:
Update strategy.js

Commit:
Update backtest.py

Commit:
Update conf-sample.js

Commit:
Update strategy.js

Commit:
Update strategy.js

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Note the y*6 + 5 at bottom

Commit:
Updated

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Update backtest.py

Commit:
Index for backtesting

Commit:
Rename index (1).php to index.php

Commit:
Delete index.php
